test/e2e/common/empty_dir.go

```diff
@@ -191,6 +192,75 @@ var _ = Describe("[sig-storage] EmptyDir volumes", func() {
 	framework.ConformanceIt("should support (non-root,0777,default) [NodeConformance]", func() {
 		doTest0777(f, testImageNonRootUid, v1.StorageMediumDefault)
 	})
+
+	It("pod should support shared volumes between containers", func() {
+		var (
+			volumeName           = "shared-data"
+			volumeMountPath      = "/usr/share/nginx/html"
+			volumeMountPath2     = "/pod-data"
```

Comment:
Can you also add variables for the file paths generated from `volumeMountPath` and use them below?